modules/oe_content_timeline_field/src/Plugin/Field/FieldWidget/TimelineFieldWidget.php

```diff
@@ -67,4 +71,51 @@ public function massageFormValues(array $values, array $form, FormStateInterface
     return $values;
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  public function errorElement(array $element, ConstraintViolationInterface $violation, array $form, FormStateInterface $form_state) {
+    if (!empty($violation->arrayPropertyPath) && $sub_element = NestedArray::getValue($element, $violation->arrayPropertyPath)) {
+      return $sub_element;
+    }
+    return $element;
+  }
+
+  /**
+   * {@inheritdoc}
+   *
+   * Override the parameters to use the form element labels.
+   */
+  public function flagErrors(FieldItemListInterface $items, ConstraintViolationListInterface $violations, array $form, FormStateInterface $form_state) {
+    /** @var \Symfony\Component\Validator\ConstraintViolationInterface $violation */
+    foreach ($violations as $offset => $violation) {
+      $initial_parameters = $violation->getParameters();
+      $new_parameters = $initial_parameters;
+      if (isset($initial_parameters['%label']) && $initial_parameters['%label'] !== 'Label') {
```

Comment:
It's good, but there's no need to check if the value is the same. Let's just replace it and then check if any diffs are available.